### ISSUE TITLE
Javadoc not built for jme3-examples

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#1366]
As noted [at the Forum](https://hub.jmonkeyengine.org/t/versions-3-3-1-and-3-3-2-of-the-engine/43140/108?u=sgold), a new Javadoc .jar file (and a corresponding .jar.asc) is deployed for each new version of most packages (in the org.jmonkeyengine repo at BinTray/JCenter) but not for jme3-examples.

To see this, browse to https://bintray.com/jmonkeyengine/org.jmonkeyengine/jme3-examples/3.3.2-stable#files/org%2Fjmonkeyengine%2Fjme3-examples%2F3.3.2-stable

This looks to be intentional. jme3-examples/gradle.properties has:
```properties
# When running this project we don't need javadoc to be built.
buildJavaDoc = false
```

The rationale isn't clear to me. Why do other subprojects need javadoc but not this one?

**Comment by stephengold** [comment]
related issue: #1370

**Comment by stephengold** [comment]
related issue: #1468 -- in order to publish jme3-examples to MavenCentral, we must provide javadoc

**Comment by stephengold** [comment]
Verified fixed in v3.4.0-alpha1. Classes from jme3-examples appear in both the v3.4.0-alpha1 merged javadoc and jme3-examples-3.4.0-alpha1-javadoc.jar
